### PR TITLE
Surrounded register and settings with try/catch

### PR DESCRIPTION
**Pull request by Olegas123**
Changes:
Register and settings now are surrounded in try/catch statements, which allows to make more readable code (because getting rid of a big amount of if's, if else's and etc. which makes code reading hard(especially at reg. page cs.) 
Changed boolean functions into a void, because now it does not return false or true - it literally returns an EXCEPTION, which are being catche, so we can have more flexible code :)
Added customexceptions file, so now we can create our own exceptions :)
Fixes:
Small insignificant fixes :)

### CODE REVIEW

File: Kinder/Registration.xaml.cs [hunk 1]
```diff
@@ -28,119 +28,155 @@ public RegistrationWindow()
...
+            }
+
+            try                                                     ///Phonenumber try/catch block
             {
-                ErrorEmail.Visibility = Visibility.Hidden;
-                Email.BorderBrush = Brushes.Green;
+                RegValidation.CheckTextBoxInput(PhoneNumber.Text);
+                RegValidation.CheckIfPhoneNumberValid(PhoneNumber.Text);
+                User.CheckIfUserAlreadyExists(PhoneNumber.Text);
+                ErrorPhoneNumber.Visibility = Visibility.Hidden;
```
Comment (JustasBan) [inline Kinder/Registration.xaml.cs:119]: isn't visibility hidden by default?

Comment (JustasBan) [reply Kinder/Registration.xaml.cs:119]: if it isn't maybe set up visibility before try block, since setting up visibility isn't "trying" to detect error

Comment (JustasBan) [reply Kinder/Registration.xaml.cs:119]: that would go for other similar sequences in other try blocks too

File: Kinder/Settings.xaml.cs [hunk 2]
```diff
@@ -27,47 +27,60 @@ public Settings()
 
         private void ChangeEmail_Click(object sender, RoutedEventArgs e)
         {
-            if (RegValidation.CheckIfEmailValid(NewEmail.Text))
+            try
             {
+                RegValidation.CheckIfEmailValid(NewEmail.Text);
                 User.ChangeUserEmail(NewEmail.Text);
                 MessageBox.Show("Email changed to " + NewEmail.Text + ".");
```
Comment (JustasBan) [inline Kinder/Settings.xaml.cs:34]: wouldn't user see message "email changed to ...", if their email was not valid?

Comment (JustasBan) [reply Kinder/Settings.xaml.cs:34]: hmm, or maybe code doesn't reach this line, when exception is thrown

Comment (Olegas123) [reply Kinder/Settings.xaml.cs:34]: When an exception is thrown, it does not go to following code, it goes straight to catch block

File: Kinder/Registration.xaml.cs (hunk 1 above)
Comment (Olegas123) [reply Kinder/Registration.xaml.cs:119]: Hm, you may be right. I moved visibility (the green one) before try block

**Pull request merged**